docs/api/en/lights/SpotLight.html

```diff
@@ -21,10 +21,11 @@ <h1>[name]</h1>
 
 		<h2>Code Example</h2>
 		<code>
-		// white spotlight shining from the side, casting a shadow
+		// white spotlight shining from the side, modulated by a texture, casting a shadow
 
 		var spotLight = new THREE.SpotLight( 0xffffff );
 		spotLight.position.set( 100, 1000, 100 );
+		spotLight.map = new THREE.TextureLoader().load(textureUrl);
```

Comment:
I would not include the usage of `map` in the basic spot light example.

Reply:
that's an editorial call, I'll remove it if needed